.github/workflows/ui_test_job.yml

```diff
@@ -62,12 +62,10 @@ jobs:
         if: inputs.run_against_image
         uses: aws-actions/configure-aws-credentials@v4.0.1
         with:
-          aws-access-key-id: ${{ secrets.aws_access_key_id }}
-          aws-secret-access-key: ${{ secrets.aws_secret_access_key }}
           aws-region: eu-west-1
-          role-to-assume: arn:aws:iam::311462405659:role/modernising-lpa-ci
+          role-to-assume: arn:aws:iam::311462405659:role/modernising-lpa-github-actions-ecr-pull
```

Comment:
Does this role need to be created somewhere (or does it already exist?)

Reply:
it's created and used in this PR's runs (https://github.com/ministryofjustice/opg-org-infra/pull/2386)